biztalk/core/how-to-modify-group-settings.md

```diff
@@ -38,10 +38,19 @@ Using the Settings Dashboard, you can modify the configuration information used
    |          **Configuration refresh interval**           |                                                                                                                        Set the interval in which [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] refreshes messaging configuration.                                                                                                                        |             1 - 43200             |       -       |                                                      -                                                      |
    |              **Message Batch Threshold**              |                                                                                                                                    If the total size of an incoming message Batch exceeds this value, then it is split into smaller batches and processed.                                                                                                                                    |           1 - 10000000            |    102400     | Copies the HKEY_LOCAL_MACHINE\Software\Microsoft\BizTalk Server\3.0\Administration\TransformThreshold value |
    |                **Large message size**                 |                                                                                                                                       Set the threshold size of an individual message that triggers streaming in a batch and/or during transformations.                                                                                                                                       |           1 - 10000000            |    1000000    |           Maximum of the existing **Large message size** and **LargeMessageFragmentSize** values.           |
+   |              **Receive Location**                     |
+   |              **Enable fault tolerance**               | New starting with BizTalk Server 2020.<br /><br />Select this option to enable receive location recover from fault. <br /><br /> Receive locations can recover from transient errors instead of completely disabled by BizTalk. On error, receive locations will be disabled in the host instance where it is faulted and BizTalk will attempt to recover it in certain configurable interval. The receive location will continue running in other host instances. |              On, Off              |      Off       |
```

Comment:
Mandi has marked settings which are new by adding the statement after the description - example below:

![image](https://user-images.githubusercontent.com/45184761/72929354-86e1c180-3d0e-11ea-99c0-4002d5352b08.png)

(https://docs.microsoft.com/en-us/biztalk/core/how-to-configure-scheduling-for-a-receive-location)

I think we should use the same way for consistency